common/script/fns/statsComputed.js

```diff
@@ -0,0 +1,28 @@
+import _ from 'lodash';
+import content from '../content/index';
+import splitWhitespace from '../libs/splitWhitespace';
+const $w = splitWhitespace;
+import * as statHelpers from '../statHelpers';
+
+module.exports = function statsComputed (user) {
+  let thisArr = $w('stats stats.buffs items.gear.equipped.weapon items.gear.equipped.armor items.gear.equipped.head items.gear.equipped.shield');
```

Comment:
But it becomes an array anyway. $w() is the array shorthand. This syntax looks very clean to me.